source/elements/oneDAL/source/data_management/tables.rst

```diff
@@ -419,8 +415,8 @@ Feature info
 
    Invariants:
       | ``feature_type::nominal`` or ``feature_type::ordinal``
-        are avaliable only with integer ``data_type``
-      | ``feature_type::contiguous`` avaliable only with floating-point ``data_type``
+        are available only with integer ``data_type``
+      | ``feature_type::contiguous`` available only with floating-point ``data_type``
 
 .. _Data type:
```

Comment:
wordings on api description:
constructor - constructs
copy constructor - copies
mov constructor - moves
etc
N->n
table_metametadata = table_meta() no space

Homogeneous table section introduces the defs again - just refer to the table above
Follows the conventions or reqs or so and ref